packages/settings-library/src/settings_library/comp_services.py

```diff
@@ -0,0 +1,25 @@
+from pydantic import ByteSize, NonNegativeInt, validator
+from settings_library.base import BaseCustomSettings
+
+_DEFAULT_MAX_NANO_CPUS_VALUE = 4 * pow(10, 9)
+_DEFAULT_MAX_MEMORY_VALUE = 2 * pow(1024, 3)  # 2 GiB
+
+
+class CompServices(BaseCustomSettings):
```

Comment:
MINOR: ``ComputationalServices`` ?

Reply:
I never know if you want short or long names ;)